requirements.txt

```diff
@@ -1,11 +1,2 @@
 -e git+https://github.com/nipy/nipype.git@master#egg=nipype
 -e git+https://github.com/incf/pybids.git@master#egg=pybids
-numpy
-lockfile
-future
-scikit-learn
-matplotlib
-nilearn
-sklearn
-nibabel
-niworkflows>=0.0.3a3
```

Comment:
Yes, I think that is one possibility. However, I don't think it makes a lot of sense: the project becomes editable mode always. The only case for which I've seen the requirements.txt is really necessary is autobuilding documentation in RTD.